lib/node_modules/@stdlib/stats/base/ndarray/nanmax-by/docs/types/index.d.ts

```diff
@@ -0,0 +1,62 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 4.1
+
+/// <reference types="@stdlib/types"/>
+
+import { typedndarray } from '@stdlib/types/ndarray';
+
+type Nullary<U> = ( this: U ) => number | void;
```

Comment:
Please refer to `stats/base/ndarray/max-by`: https://github.com/stdlib-js/stdlib/blob/ff45992200d325437e5216d0e7738ee812c661dc/lib/node_modules/%40stdlib/stats/base/ndarray/max-by/docs/types/index.d.ts#L25. We expect all functions to have TSDoc comments.